api/Makefile

```diff
@@ -70,26 +66,16 @@ test: lint tests
 lint:
 	$(RUNNER) flake8 .
 
-tests: PIPENV_DOTENV_LOCATION := $(TEST_PIPENV_DOTENV_LOCATION)
 tests: version-file
-	$(MAKE) setup-test-db
```

Comment:
Do we still need the target `setup-test-db` now that you've remove their invocation?

Reply:
yes will do in a PR later as discussed